.gitlab/source_test/ebpf.yml

```diff
@@ -58,8 +58,17 @@
   script:
     - $ErrorActionPreference = "Stop"
     - '$_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content ; Write-Host "Running on instance $($_instance_id)"'
-    - !reference [.setup_python_mirror_win]
-    - docker run --rm -m 8192M -v "$(Get-Location):c:\mnt" -e AWS_NETWORKING=true -e SIGN_WINDOWS=true -e PY_RUNTIMES="$PYTHON_RUNTIMES" -e GOMODCACHE="c:\modcache" -e PIP_INDEX_URL=${PIP_INDEX_URL} 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/windows_1809_${ARCH}:$Env:DATADOG_AGENT_WINBUILDIMAGES c:\mnt\tasks\winbuildscripts\sysprobe.bat
+    - >-
```

Comment:
i think you need just the `>` character

Reply:
Right, fixed in [1949156](https://github.com/DataDog/datadog-agent/pull/14899/commits/1949156bb0f6750def2e4aeae939f2c62b18e160) ! Thanks !